tailwind.config.js

```
@@ -1,6 +1,8 @@
 import typography from '@tailwindcss/typography';
 
 /** @type {import('tailwindcss').Config} */
+import typography from '@tailwindcss/typography';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'typography'. Consider to delete it or rename it.

'typography' is defined here:


(lint/suspicious/noRedeclare)